<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="placeholderText">
         <string>Search packages</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="searchModeComboBox">
        <item>
         <property name="text">
          <string>Repo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AUR</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="searchByComboBox">
        <item>
         <property name="text">
          <string>Name, description</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Description</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="PackagesView" name="packagesView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="packageHeadLayout">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="packageTitleLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="packageTabsWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="infoTab">
       <property name="loaded" stdset="0">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="infoScrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="infoContentWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>707</width>
             <height>365</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="infoContentLayout">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="archTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Architecture:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="archLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="repoTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Repo:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="repoLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="urlTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>URL:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="urlLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="licensesTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Licenses:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="licensesLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="groupsTitlelabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Groups:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="groupslabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="downloadSizeTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Download size:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="downloadSizeLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="installedSizeTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Installed size:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="installedSizeLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="maintainerTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Maintainer:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="maintainerLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="buildDateTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Build date:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLabel" name="buildDateLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="installDateTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Install date:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLabel" name="installDateLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="reasonTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Installation reason:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLabel" name="reasonLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="scriptTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Installation script:</string>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QLabel" name="scriptLabel">
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="depsTab">
       <property name="loaded" stdset="0">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Dependencies</string>
       </attribute>
       <layout class="QVBoxLayout" name="depsTabLayout">
        <item>
         <widget class="QScrollArea" name="depsScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="depsContentWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>707</width>
             <height>365</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="depsContentLayout">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="providesTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Provides:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="providesLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="replacesTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Replaces:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QVBoxLayout" name="replacesLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="conflictsTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Conflicts:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QVBoxLayout" name="conflictsLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="dependsTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Depends:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QVBoxLayout" name="dependsLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="optdependsTitleLabel">
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>Optdepends:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QVBoxLayout" name="optdependsLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filesTab">
       <property name="loaded" stdset="0">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QVBoxLayout" name="filesTabLayout">
        <item>
         <widget class="FilesView" name="filesView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FilesView</class>
   <extends>QTreeView</extends>
   <header>src/files-view/filesview.h</header>
  </customwidget>
  <customwidget>
   <class>PackagesView</class>
   <extends>QTreeView</extends>
   <header>src/packages-view/packagesview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>searchByComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_searchEdit_returnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_searchEdit_returnPressed()</slot>
 </slots>
</ui>
